src/components/admin/AdminContentList.tsx

```diff
@@ -103,7 +112,26 @@ const AdminContentList: React.FC<Props> = props => {
 
   return (
     <>
-      <Title level={3}>{props.title}</Title>
+      <Flex>
+        <Title level={3}>{props.title}</Title>
+      </Flex>
+
+      {(props.title=="Categories") && (
+        <Text pb={2}>
+          Categories are prizes or awards that hackathon submissions can win. 
+          For example, “Best Overall”  or “T-Mobile Winner” or “Best Design”. Categories belong 
+          to category groups for judging organization purposes.
+        </Text>
+      )}
```

Comment:
I think keeping this here is enough. Everything else would be a little too much.